.github/workflows/build_library.yml

```diff
@@ -0,0 +1,33 @@
+name: Binary Release Build
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+  
+    steps:
+      - uses: actions/checkout@v2
+    
+      - name: Set up docker for build (x86_64)
+        # Add more of these steps to support other architectures
```

Comment:
We can use a build matrix for this :)

Reply:
![Screen Shot 2021-04-14 at 6 03 25 PM](https://user-images.githubusercontent.com/1626430/115425096-257bfd00-a1b4-11eb-9d62-433c4636fe95.png)

Well, I managed to build this, together with a lot of other common architectures like MIPS and i386 but I did not have access to an IBM S/390 mainframe to test if it worked :D